confidential-identity/wasm/examples/index.test.js

```diff
@@ -0,0 +1,101 @@
+const cil = require('../pkg/');
+const crypto = require('crypto');
+
+
+let investor_did = [0x49, 0x99, 0x52, 0x43, 0x74, 0x8c, 0x4a, 0xe7, 0x11, 0x8, 0x3c, 0x97, 0x56, 0x5f, 0xfd, 0xfd, 0x60, 0xdb, 0x1d, 0x8c, 0xc5, 0x85, 0xf8, 0xa7, 0x1d, 0x99, 0x93, 0x9c, 0xbe, 0xab, 0xdd, 0x5];
```

Comment:
if these never change you could use `const` instead of `let`

Reply:
Sure, will do